api/workAllocation2/util.ts

```diff
@@ -500,3 +506,8 @@ export async function getTypesOfWorkByUserId(path, req: express.Request): Promis
   }
   return null;
 }
+
+export function getUniqueCases(caseData: RoleCaseData[]): number {
```

Comment:
This should be called `getUniqueCasesCount`

Reply:
sure